src/overlays/actors/ovl_Bg_Kin2_Picture/z_bg_kin2_picture.h

```diff
@@ -3,17 +3,37 @@
 
 #include "global.h"
 
+#define BG_KIN2_PICTURE_GET_1F(thisx) ((thisx)->params & 0x1F)
+#define BG_KIN2_PICTURE_GET_PARAMS_05(thisx) (((thisx)->params >> 5) & 1)
+
 struct BgKin2Picture;
 
 typedef void (*BgKin2PictureActionFunc)(struct BgKin2Picture*, PlayState*);
 
 typedef struct BgKin2Picture {
-    /* 0x000 */ Actor actor;
-    /* 0x144 */ char unk_144[0xF0];
+    /* 0x000 */ DynaPolyActor dyna;
+    /* 0x15C */ ColliderTris colliderTris;           /* inferred */
```

Comment:
You can delete all this `/* inferred */` stuff; it comes from M2C and doesn't mean anything outside of that.